tests/analysis_new_tests/tests/test.js.snapshot

```diff
@@ -0,0 +1,7 @@
+exports[`RecordFieldCompletions.res - Record field completion in nested record 1`] = `
```

Comment:
I know you didn't focus on this, but this really is an unreadable snapshot file.